src/js/controllers/mapController.ts

```diff
@@ -1183,6 +1188,289 @@ export class MapController {
+            const specificLayer = this._map?.findLayerById(layerID);
+            if (specificLayer) {
+              if (specificLayer.id.includes('72')) {
+                specificLayer.visible = true;
+              } else {
+                specificLayer.visible = false;
+              }
+            }
+          });
+        }
+        break;
+      case '7 days':
+        {
+          MODIS24.visible = false;
+          MODISLayerIDs.forEach(({ layerID }) => {
+            const specificLayer = this._map?.findLayerById(layerID);
+            if (specificLayer) {
+              if (specificLayer.id.includes('7d')) {
+                specificLayer.visible = true;
+              } else {
+                specificLayer.visible = false;
+              }
+            }
+          });
+        }
+        break;
+      default:
+        break;
+    }
+  }
+
+  setVIIRSDefinedRange(layer: any, sublayerType: string): void {
+    if (!this._map) {
+      return;
+    }
+    const VIIRS24 = layer.sublayers.items.filter((sublayer: Sublayer) =>
+      sublayer.title.includes('24 hrs')
+    );
+
+    switch (sublayerType) {
+      case '24 hrs':
+        {
+          VIIRS24.visible = true;
+          VIIRSLayerIDs.forEach(({ layerID }) => {
+            const specificLayer = this._map?.findLayerById(layerID);
+
+            if (specificLayer) {
+              specificLayer.visible = false;
+            }
+          });
+        }
+        break;
+      case '48 hrs':
+        {
+          VIIRS24.visible = false;
+          VIIRSLayerIDs.forEach(({ layerID }) => {
+            const specificLayer = this._map?.findLayerById(layerID);
+            if (specificLayer) {
+              if (specificLayer.id.includes('48')) {
+                specificLayer.visible = true;
+              } else {
+                specificLayer.visible = false;
+              }
+            }
+          });
+        }
+        break;
+      case '72 hrs':
+        {
+          VIIRS24.visible = false;
+          VIIRSLayerIDs.forEach(({ layerID }) => {
+            const specificLayer = this._map?.findLayerById(layerID);
+            if (specificLayer) {
+              if (specificLayer.id.includes('72')) {
+                specificLayer.visible = true;
+              } else {
+                specificLayer.visible = false;
+              }
+            }
+          });
+        }
+        break;
+      case '7 days':
+        {
+          VIIRS24.visible = false;
+          VIIRSLayerIDs.forEach(({ layerID }) => {
+            const specificLayer = this._map?.findLayerById(layerID);
+            if (specificLayer) {
+              if (specificLayer.id.includes('7d')) {
+                specificLayer.visible = true;
+              } else {
+                specificLayer.visible = false;
+              }
+            }
+          });
+        }
+        break;
+      default:
+        break;
+    }
+  }
+
+  resetDefinedDateRange(layerID: string): void {
```

Comment:
can we name this method something like `turnOffFireLayers`, I may be missing something but loooks like we are just turning off visibility but our method is saying it is resetting defined date range?

Reply:
@vaidotasp will do - we're turning off the visibility of the 24hr, 48hr, 72hr and 7day sublayer/feature layers for `VIIRS` or `MODIS`